.ci/scripts/build.sh

```diff
@@ -1,6 +1,7 @@
 #! /bin/bash
 #===============================================================================
 # Copyright 2019 Intel Corporation
+# Copyright 2023-24 FUJITSU LIMITED
```

Comment:
Per discussion with UXL - we can start adding "Copyright Contributors to the oneDAL project", while keeping existing Intel copyright that already exist. 

There should be some checks fixes done internally to reflect this + we will be starting using new copyright as well